x/shared/types/params_test.go

```diff
@@ -190,3 +190,33 @@ func TestParams_ValidateGracePeriodEndOffsetBlocks(t *testing.T) {
 		})
 	}
 }
+
+func TestParams_ValidateSupplierUnbondingPeriod(t *testing.T) {
```

Comment:
Should we add validation for `validateSupplierUnbondingPeriodIsGreaterThanCumulatedProofWindowCloseBlocks` here too?